lib/simulation/openfisca/mapping/reverse.js

```diff
@@ -15,30 +15,31 @@ module.exports = function reverseMap(openfiscaResponse, situation) {
     // Computed prestations can be in the 'familles' or 'individus' openfisca entity
     var openfiscaResults = _.merge(openfiscaResponse.value[0].familles[0], openfiscaResponse.value[0].individus[0]);
 
-    return _.mapValues(PRESTATIONS, function(format, prestationName) {
-
-        // For prestations that have been injected by the user and not calculated by the simulator, put null to allow specific treatment in the ui
-        if (_.contains(injectedPrestations, prestationName)) {
-            return null;
-        }
-
-        var type = format.type,
-            computedPrestation = openfiscaResults[prestationName],
-            result = computedPrestation[period];
-
-        var uncomputabilityReason = openfiscaResults[prestationName + '_non_calculable'] && openfiscaResults[prestationName + '_non_calculable'][period];
-
-        if (uncomputabilityReason) {
-            return uncomputabilityReason;
-        }
-
-        if (format.montantAnnuel) {
-            result *= 12;
-        }
-
-        if (type == Number) {
-            result = Number(result.toFixed(2));
-        }
-        return result;
+    var calculatedPrestations = _.pick(PRESTATIONS, function(format, prestationName) {
```

Comment:
Since lodash 4.0 yes, but before `pick` had [this behaviour] (https://github.com/lodash/lodash/blob/3.10.1/doc/README.md#_pickobject-predicate-thisarg).

Today we would use [`pickBy`](https://lodash.com/docs#pickBy).

`PRESTATIONS` is an object so there is no `filter` native method. And `_.filter` returns an array, which is not what we want.